lib/Page.js

```diff
@@ -502,6 +533,18 @@ Page.prototype.generate = function (builtFiles) {
   });
 };
 
+/**
+ * Adds linked layout files to page assets
+ */
+Page.prototype.addLayoutFiles = function () {
+  this.asset.layoutScript = path.relative(path.dirname(this.resultPath),
+                                          path.join(this.rootPath, LAYOUT_FOLDER_PATH,
+                                                    this.frontMatter.layout, 'scripts.js'));
```

Comment:
These files need to be copied over to the actual site folder, and the path should reference that, not the ones in `_markbind`, which will not be in the final site files. Their path all refer to `_markbind` atm.